arc/src/arc_curie_pme.c

```diff
@@ -20,11 +20,12 @@ void curie_pme_begin(void)
     reg_write16(NSR, saved_nsr);
 }
 
-void curie_pme_configure(u16_t global_context,
-                         PATTERN_MATCHING_DISTANCE_MODE distance_mode,
-                         PATTERN_MATCHING_CLASSIFICATION_MODE classification_mode,
-                         u16_t min_if,
-                         u16_t max_if)
+void curie_pme_configure(
+    u16_t global_context,
+    PATTERN_MATCHING_DISTANCE_MODE distance_mode,
+    PATTERN_MATCHING_CLASSIFICATION_MODE classification_mode,
+    u16_t min_if,
+    u16_t max_if)
```

Comment:
I am specifically telling clang to try to use 1 arg per line when the whole line doesn't fit, only in the function declarations, but not when calling the functions.  Since I've seen how clang treats it differently on different places, I rather enforce the rules to have a consistent style so I can just run the script without going back to manually revert the style, but if you prefer the fit everything you can on the same line, I am okay with it.